LLM-Translation.popclipext/Config.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "LLM Translation",
-  "icon": "icon.svg",
+  "icon": "translation.svg",
```

Comment:
The translation.svg is too complex to display in the PopClip menu, so I use a simpler version. Does it have any issues?

Reply:
It shows up as a solid disk.

<img width="157" alt="Screenshot 2025-04-07 at 8 40 36 AM" src="https://github.com/user-attachments/assets/b2060486-6690-45ae-88cc-2ca7203c02a7" />

Reply:
![CleanShot 2025-04-07 at 21 51 48](https://github.com/user-attachments/assets/9e479c9e-1ec9-40b9-8103-9a161fc53be1)

Reply:
I reinstalled and it is fine this time ...